config/initializers/gds_api.rb

```diff
@@ -0,0 +1,3 @@
+require 'gds_api/base'
+
+GdsApi::Base.logger = Logger.new(Rails.root.join("log/#{Rails.env}.api_client.log"))
```

Comment:
Oh ok but what I mean is how will we see these logs? Is this just gonna be used in development? I'm scared that we unwittingly use up all the disk space on some production server. Logs get big quiiiick:
```
[~/.../service-manual-publisher] master ∞ du -h --max-depth 1
544K	./coverage
180K	./db
20K	./bin
32M	./.git
16K	./lib
53M	./tmp
292K	./spec
20K	./vendor
918M	./log
108K	./config
20K	./public
456K	./app
1004M	.

```